piquasso/_simulators/fock/state.py

```diff
@@ -202,6 +203,50 @@ def wigner_function(
 
         return 0.5 * W * g**2
 
+    def plot_wigner(
+        self,
+        positions: List[float],
+        momentums: List[float],
+        mode: Optional[Tuple[int, ...]] = None,
+    ) -> None:
+        r"""
+        Plots the Wigner function in phase space for a single mode using the
+        given positions and momentums.
+
+        Args:
+            positions (List[float]): List of position values (x-axis)
+            momentums (List[float]): List of momentum values (p-axis)
+            mode (tuple[int], optional):
+                Mode where Wigner function should be calculcated.
+
+        Note:
+            Only a single mode is supported.
+
+        Returns:
+            None: This method generates a plot and does not return a value.
+
+        """
+        if self._config.validate and (
+            self.d != 1 or (mode is not None and len(mode) != 1)
+        ):
+            raise InvalidModes(
+                "The Wigner function can only be calculated for a single mode: "
```

Comment:
Just saw this - we can calculate the Wigner function for multiple modes, but can only plot one mode in 2D!
```suggestion
                "The Wigner function can only be plotted for a single mode: "
```